RevenueCat/Scripts/LogLevel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.ComponentModel;
+using static Purchases;
+
+public partial class Purchases
+{
+    /// <summary>
+    /// Enum of available log levels.
+    /// </summary>
+    public enum LogLevel
+    {
+        [Description("VERBOSE")]
+        Verbose,
+        [Description("DEBUG")]
+        Debug,
+        [Description("INFO")]
+        Info,
+        [Description("WARN")]
+        Warning,
```

Comment:
Should this be `Warn`? Or is it more common in Unity to name it Warning?

Reply:
C# is usually pretty verbose, but I have no idea what's more natural. I can change it if you think that'd be better.

Reply:
I actually prefer `Warning` over `Warn`. But I also don't have an idea of what's more natural. Maybe it should be warn just because the other SDKs do Warn too?

Reply:
Ok I'll change it.